tools/autograph-client/client.go

```diff
@@ -71,6 +90,8 @@ examples:
 	U2lnbmF0dXJlLVZlcnNpb246IDEuMApNRDUt...
 	$ go run client.go -d U2lnbmF0dXJlLVZlcnNpb2...  -cn cariboumaurice -k webextensions-rsa -o detachedxpisig.pkcs7
 
+* sign an XPI file:
+        $ go run client.go -f unsigned.xpi -cn cariboumaurice -k webextensions-rsa -o signed.xpi
```

Comment:
in case you wonder where `cariboumaurice` comes from: https://www.dailymotion.com/video/xtddv

Reply:
reminds me of this play https://www.youtube.com/watch?v=kqJo51vlZOU